library/HelloSign/Client.php

```diff
@@ -495,15 +495,14 @@ public function getSignatureRequest($id)
      * Retrieves the current user's signature requests. The resulting object
      * represents a paged query result.
      *
-     * @param  Integer $page
+     * @param  Array of search parameters ['query'=>'to:user@email.com']
      * @return SignatureRequestList
      * @throws BaseException
      */
-    public function getSignatureRequests($page = 1)
+    public function getSignatureRequests($params = [])
     {
-        $params = array(
-            'page' => $page
-        );
+        if(!$params['page'])
```

Comment:
it's always a good idea to use `if(isset($params['page']) && !$params['page'])` because php will throw notices in the logs if you don't specifically check that keys exist. This is not javascript where is an object exists and you test a key, it won't complain about it.